gpu_opt.py

```
@@ -0,0 +1,23 @@
+"""GPU utils"""
+import subprocess,os,hashlib,urllib.request,tempfile,time,threading
+_W=False
+_L=threading.Lock()
+def _g():
+ global _W
+ with _L:
+  if _W:return
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)